_includes/components-repositories.md

```diff
@@ -4,28 +4,21 @@ Current set of components:
 _Information on new components can be found on our [project roadmap]({{ site.baseurl }}/roadmap/)._  
 
 **cf-buttons:**
-[repo](https://github.com/cfpb/cf-buttons) /
-[style guide](http://cfpb.github.io/cf-buttons/docs/)  
+[repo](https://www.npmjs.com/package/cf-buttons)
```

Comment:
Yes but I don't think this file is actually used any more. Deleting it and adding npm badges to the components page. Thanks!